Sources/UnidirectionalBinding.swift

```diff
@@ -276,3 +277,62 @@ extension BindingTarget where Value: OptionalProtocol {
 		return target <~ property.producer
 	}
 }
+
+/// A pseudo `BindingTarget`, constructed from a closure and a lifetime.
+public final class AnyBindingTarget<Value>: BindingTarget {
+	public let lifetime: Lifetime
+	private let setter: (Value) -> Void
+
+	/// Wraps a `BindingTarget`.
+	///
+	/// - parameters:
+	///   - target: The binding target to wrap.
+	public init<T: BindingTarget>(_ target: T) where T.Value == Value {
+		lifetime = target.lifetime
+		setter = target.consume
```

Comment:
I have removed it, since it isn't needed now for the Rex merger.

Anyway, implementing this correctly would probably require type erasing via inheritance and runtime type check, especially for the `<~` requirement.